autoload/jedi.vim

```diff
@@ -392,20 +461,61 @@ endfunction
 " helper functions
 " ------------------------------------------------------------------------
 
-function! jedi#add_goto_window(len) abort
-    set lazyredraw
-    cclose
+" a:mode: 'goto' or 'usages'
```

Comment:
Consider making this a boolean. There's quite a bit of confusion already around the different goto variants (stubs, assignments, definitions, etc.). `is_usages` would probably be way clearer.

Reply:
Good point. Used `for_usages`.